doc/datasets/toy_dataset.rst

```diff
@@ -9,10 +9,10 @@ Toy datasets
 
 .. currentmodule:: sklearn.datasets
 
-scikit-learn comes with a few small standard datasets that do not require to
-download any file from some external website.
+scikit-learn comes preloaded with a few small, well-known datasets, which allows you to
+experiment without downloading external files.
```

Comment:
I am -1 for this change. "preloaded" does not look correct and I don't see an improvement for the remaining of the sentence.